queries.sql

```diff
@@ -88,11 +88,11 @@ SELECT AVG(escape_attempts)::NUMERIC(10,2) from animals WHERE EXTRACT(YEAR FROM
 /*What animals belong to Melody Pond?*/
 SELECT name FROM animals INNER JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
 /*List of all animals that are pokemon (their type is Pokemon).*/
-SELECT animals.name FROM animals INNER JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+SELECT * FROM animals INNER JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
 /*List all owners and their animals, remember to include those that don't own any animal.*/
 SELECT full_name, name FROM owners LEFT JOIN animals ON animals.owner_id = owners.id;
 /*How many animals are there per species?*/
-SELECT COUNT(*), species.name FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.name;
+SELECT COUNT(*) as animals_per_species, species.name FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.name;
```

Comment:
- [ ] Great job so far @Kanyorok However, just as the previous reviewer mentioned, your queries are still returning not necessarily the wrong values but incomplete values. There have to be four animals that belong to the `Pokemon` species but yours returns only 3 animals. So I think when you were inserting the values you missed one hence the issue. So kindly fix this :wink: 
- [ ] After that your first query should return `4 Pokemon animals` and the second query will also return the value `4 for Pokemon` instead of `3 for Pokemon` as it has now and `6 for Digimon` :100: